scoring/static/scoring/scss/typo.scss

```diff
@@ -1,4 +1,4 @@
-body {
+body, .body {
```

Comment:
Where does a class of "body" get used? I can’t find it anywhere.

Reply:
This is been used on the `.hidden-message ` on the utilities file.